### ISSUE TITLE
How to silence warnings about notebooks without run commands

### ISSUE THREAD

**Issue by habemus-papadum** [Anaconda-Platform/anaconda-project#263]
Hello, 
When I run anaconda-project, I see warnings such as `anaconda-project.yml: No commands run notebooks notebooks/foo.ipynb, etc. `

This warning is not helpful as it does not really match my use case.  Is there a way to silence it? 

In general it would be nice to have a "quiet" mode that does not try to write output unless there is an explicit error. 

thanks!

**Comment by jlstevens** [comment]
Agreed, this warning can be alarming even when nothing is wrong and you just want to use the simplicity of:

```
commands:
  notebooks:
    notebook: .
```

**Comment by AlbertDeFusco** [comment]
Would anyone be opposed to just removing the warning all together?

**Comment by jlstevens** [comment]
Personally, I've never found it useful.

**Comment by AlbertDeFusco** [comment]
I've found a work-around while I get the next version out. Place this in your anaconda-project.yml

```yaml
skip_imports:
  notebooks: True
```

**Comment by jlstevens** [comment]
The workaround does the job, thanks!

This approach is fine though 1) it is not documented yet 2) `skip_imports` is a rather strange name for what it does here (silencing warnings).

**Comment by AlbertDeFusco** [comment]
Part of the reason for this I found in the code is that there are some cases (likely completely unused now) where anaconda-project would add a notebook command automatically.

**Comment by mcg1969** [comment]
Thanks for opening this issue, and apologies for the long delay in responding.

Anaconda Project has not been actively developed in some time and has largely been superseded by newer alternatives:

* [pixi](https://pixi.sh/) and [uv](https://docs.astral.sh/uv/) represent the first truly successful successors.
* [conda-workspaces](https://github.com/conda-incubator/conda-workspaces) is a promising pixi-compatible alternative for users who prefer to stay closer to the conda ecosystem.

Going forward, work on this repository is focused on critical bug fixes and on tooling to help users migrate existing Anaconda Project projects to pixi (and, by extension, conda-workspaces). See the [Pixi support](https://anaconda-project.readthedocs.io/en/latest/pixi-support.html) page in our documentation for details on the `anaconda-project export-pixi` command, which can convert an existing `anaconda-project.yml` into a `pixi.toml` for you.

The `anaconda-project` package itself will remain available, so existing projects relying on its stable feature set can continue to be supported. However, we do not anticipate further enhancement work, and for that reason we are closing out the bulk of our open issue backlog — including this one.

If this issue describes a critical regression in the stable feature set (as opposed to an enhancement or an edge case in a feature we no longer plan to expand) and you have a minimal reproducer against a current release, please feel free to reopen and we will take another look.

Thank you for your interest in Anaconda Project.
